### ISSUE TITLE
Gui Changes consume too much real estate

### ISSUE THREAD

**Issue by rneuber1** [erdomke/InnovatorAdmin#115]
The theme changes introduced by the latest commits change the GUI in a way that makes it harder to use, it is way too "heavy" due to the large, colored title bars etc.

It uses more real estate for the TOC (let alone its header).
It starts up with no tabs visible, you have to run a query for the "Code/Report/Table" tabs to appear.

In general, the previous non-flar design looked more professional and technical (which is the intended audience for such a tool). ;-)

![image](https://cloud.githubusercontent.com/assets/6908002/12195024/ab2876a6-b5f4-11e5-983a-4240cc42a23b.png)

**Comment by erdomke** [comment]
While I am not relenting on all the changes [;-)], I have made some changes for the next release which should handle your biggest concerns.  In particular, I decreased the font size and tweaked the header so that the overall height is the same as before.  In addition, the color is now used to render just the drop shadow and the form icons.  The still seems to be enough to make the windows distinctive (even during Alt+Tab) without being overwhelming.

The tabs are invisible by default because they serve no use until you run a query.  My overall goal is to make the user interface simple and approachable for individuals that are more in the scripting/admin camp who aren't necessarily programmers.  In addition, I am trying to balance the use cases of someone who wants to use the import/export functionality right away vs. someone who uses the tool primarily as an AML editor.  Hopefully, the new compromise achieves that.

**Comment by rneuber1** [comment]
Hi Eric,

the recent theme changes were good, it looks much cleaner now, I really like it now ;-)

\Ralf


Von: Eric Domke [mailto:notifications@github.com]
Gesendet: Freitag, 8. Januar 2016 19:12
An: erdomke/InnovatorAdmin
Cc: Neubersch, Ralf
Betreff: Re: [InnovatorAdmin] Gui Changes consume too much real estate (#115)


While I am not relenting on all the changes [;-)], I have made some changes for the next release which should handle your biggest concerns. In particular, I decreased the font size and tweaked the header so that the overall height is the same as before. In addition, the color is now used to render just the drop shadow and the form icons. The still seems to be enough to make the windows distinctive (even during Alt+Tab) without being overwhelming.

The tabs are invisible by default because they serve no use until you run a query. My overall goal is to make the user interface simple and approachable for individuals that are more in the scripting/admin camp who aren't necessarily programmers. In addition, I am trying to balance the use cases of someone who wants to use the import/export functionality right away vs. someone who uses the tool primarily as an AML editor. Hopefully, the new compromise achieves that.

—
Reply to this email directly or view it on GitHub<https://github.com/erdomke/InnovatorAdmin/issues/115#issuecomment-170077946>.

Experience the GETRAG History Web Special!
http://history.getrag.com<http://history.getrag.com/?lang=en>


________________________________

GETRAG FORD Transmissions GmbH
Firmensitz • Scarletallee 2 • 50735 Köln • Deutschland
Sitz der Gesellschaft: Köln • Registergericht Köln • HRB 34097
Geschäftsführung: Mihir Kotecha (CEO), José Ignacio Alonso, Didier Lexa, John McDonald, Susan Slaughter
Vorsitzender des Aufsichtsrats: Johannes-Jürgen Albus


Disclaimer:
Wie Sie wissen, können über das Internet versandte E-Mails unter fremden Namen erstellt oder manipuliert werden. Aus diesem Grund sind unsere per E-Mail verschickten Nachrichten grundsätzlich keine rechtsverbindlichen Willenserklärungen.
Der Inhalt dieser E-Mail ist vertraulich und ausschließlich für den bezeichneten Adressaten bestimmt. Wenn Sie nicht der vorgesehene Adressat dieser E-Mail oder dessen Vertreter sein sollten, so beachten Sie bitte, dass jede Form der Kenntnisnahme, Veröffentlichung, Vervielfältigung oder Weitergabe des Inhalts dieser E-Mail unzulässig ist. Wir bitten Sie, sich in diesem Fall mit dem Absender der E-Mail in Verbindung zu setzen.
